src/main/java/teammates/ui/webapi/action/EditFeedbackResponseCommentAction.java

```diff
@@ -0,0 +1,115 @@
+package teammates.ui.webapi.action;
+
+import java.time.Instant;
+import java.util.ArrayList;
+import java.util.List;
+
+import org.apache.http.HttpStatus;
+
+import teammates.common.datatransfer.FeedbackParticipantType;
+import teammates.common.datatransfer.attributes.FeedbackResponseAttributes;
+import teammates.common.datatransfer.attributes.FeedbackResponseCommentAttributes;
+import teammates.common.datatransfer.attributes.FeedbackSessionAttributes;
+import teammates.common.datatransfer.attributes.InstructorAttributes;
+import teammates.common.exception.EntityDoesNotExistException;
+import teammates.common.exception.EntityNotFoundException;
+import teammates.common.exception.InvalidParametersException;
+import teammates.common.util.Assumption;
+import teammates.common.util.Const;
+import teammates.ui.webapi.output.FeedbackResponseCommentData;
+
+/**
+ * Edits a feedback response comment.
+ */
+public class EditFeedbackResponseCommentAction extends Action {
+
+    @Override
+    protected AuthType getMinAuthLevel() {
+        return AuthType.LOGGED_IN;
+    }
+
+    @Override
+    public void checkSpecificAccessControl() {
+        String courseId = getNonNullRequestParamValue(Const.ParamsNames.COURSE_ID);
+        String feedbackSessionName = getNonNullRequestParamValue(Const.ParamsNames.FEEDBACK_SESSION_NAME);
+        String feedbackResponseId = getNonNullRequestParamValue(Const.ParamsNames.FEEDBACK_RESPONSE_ID);
+        Long feedbackResponseCommentId = getLongRequestParamValue(Const.ParamsNames.FEEDBACK_RESPONSE_COMMENT_ID);
```

Comment:
The `courseId`, `feedbackSessionName` and `feedbackResponseId` are not necessary as they all stored in `FeedbackResponseAttributes`? Same for Create/Delete action.